packages/bolt-interactions/README.md

```diff
@@ -0,0 +1,189 @@
+# Bolt ⚡️ Interactions
+
+Bolt interactions allows you to effortlessly create stateful user action `flows` with your [Bolt](https://github.com/slackapi/bolt) app.
+
+Vanilla Bolt gives you some ability to do this already, but it's state is scoped to a whole channel. This library keeps the scope to the interaction allowing for faster data fetching and a greater ability to rationalize how your app works.
+
+## Install
+
+```bash
+yarn add @slack-wrench/bolt-interactions
+# or
+npm install --save @slack-wrench/bolt-interactions
+```
+
+## Understanding Flows
+
+Bolt interactions creates a new concept of a user interaction called a `flow`. A flow is set of bolt event listeners that are linked by a shared state.
+
+Here's an example:
+
+```typescript
```

Comment:
That sounds like a lot of code. I was honestly pretty worried about dropping _this much_ code as simple example.

Assuming this code actually makes things simpler 🤞 the before flow will be way worse. I think leaving it as is is fine?

Reply:
I'm fine with this - I think it might be hard for people to visualize the use case as is, maybe readme just needs another pass. People who get it will get it, which is probably good enough for now.